process-compose-mp.yml

```diff
@@ -4,67 +4,111 @@ environment:
   - ESPRESSO_SEQUENCER_GENESIS_FILE=data/genesis/demo-marketplace.toml
   - ESPRESSO_BUILDER_GENESIS_FILE=data/genesis/demo-marketplace.toml
 processes:
+
+  marketplace-reserve-builder:
```

Comment:
I had to move these here because the builders were disabled. seems like disabled:false setting is not working for overridden file